ui/apps/pixano/src/components/layout/DatasetHeader.svelte

```diff
@@ -105,22 +105,25 @@
         <button on:click={() => navigateTo("/")} class="h-10 w-10">
           <img src={pixanoLogo} alt="Logo Pixano" class="w-8 h-8 mx-2" />
         </button>
-        <IconButton on:click={() => navigateTo("/")}>
+        <IconButton
+          on:click={() => navigateTo(currentItemId ? `/${datasetName}/dataset` : "/")}
```

Comment:
I guess an actual "Go back" button is not really necessary since we can use the browser's own "Go back" and "Go forward" buttons now. 

I made this change because the button is included in the Sketch template, but otherwise I would have considered deleting it, since like you said there is the logo and the dedicated button. Then again having that button is maybe more intuitive for users. This probably deserves a discussion with Janro.